src/main/java/io/anserini/ltr/TwitterFeatureReranker.java

```diff
@@ -39,7 +39,7 @@ public ScoredDocuments rerank(ScoredDocuments docs, RerankerContext context) {
         continue;
       }
 
-      String qid = ((String)context.getQueryId()).replaceFirst("^MB0*", "");
+      String qid = (context.getQueryId()).replaceFirst("^MB0*", "");
```

Comment:
`int qid = Integer.valueOf(_)`

Reply:
Since we are going to use `Integer` for `context.getQueryId()` here, can we just do: `int qid = context.getQueryId();`? I'm assuming the `Integer` value of the query id won't contain `MB0`.

Reply:
Yes, you are correct.